sdk/storage/storage-blob/src/index.browser.ts

```diff
@@ -37,3 +37,17 @@ export {
   PageList
 } from "./PageBlobRangeResponse";
 export { logger } from "./log";
+
+export function configureFallback(
+  domParser: { new (): DOMParser },
+  xmlSerializer: { new (): XMLSerializer },
+  document: Document,
+  node: { new (): Node }
+) {
+  if (!self) {
```

Comment:
What other changes should I do?

- If `window` is defined maybe we don't allow fallbacks?
- spread operator didn't quite work for me here because my Node type is missing the required properties... any ideas what I can do there?
The error I got was:

```
Type '{ new (): Node; TEXT_NODE: number; prototype: Node; }' is missing the following properties from type '{ new (): Node; prototype: Node; readonly ATTRIBUTE_NODE: number; readonly CDATA_SECTION_NODE: number; readonly COMMENT_NODE: number; readonly DOCUMENT_FRAGMENT_NODE: number; ... 13 more ...; readonly TEXT_NODE: number; }': ATTRIBUTE_NODE, CDATA_SECTION_NODE, COMMENT_NODE, DOCUMENT_FRAGMENT_NODE, and 13 more.ts(2740)
```

I'm sure I'm just declaring the types incorrectly, but rather than wrestle with it I figured I'll ask if someone here knows how to declare the type correctly...